File: domain/src/main/java/no/nav/melosys/domain/dokument/arbeidsforhold/Arbeidstidsordning.java
package no.nav.melosys.domain.dokument.arbeidsforhold;

import no.nav.melosys.domain.FellesKodeverk;
import no.nav.melosys.domain.dokument.felles.AbstraktKodeverkHjelper;

public class Arbeidstidsordning extends AbstraktKodeverkHjelper {
    @Override
    public FellesKodeverk hentKodeverkNavn() {
        return FellesKodeverk.ARBEIDSTIDSORDNINGER;
    }

}

File: domain/src/main/kotlin/no/nav/melosys/domain/dokument/arbeidsforhold/Arbeidstidsordning.kt
package no.nav.melosys.domain.dokument.arbeidsforhold

import no.nav.melosys.domain.FellesKodeverk
import no.nav.melosys.domain.dokument.felles.AbstraktKodeverkHjelper



class Arbeidstidsordning : AbstraktKodeverkHjelper() {
    override fun hentKodeverkNavn(): FellesKodeverk = FellesKodeverk.ARBEIDSTIDSORDNINGER
}
